### PR TITLE
docs: Update CANalyst-II interface describe (#179)

### PR DESCRIPTION
**Pull request by Cherish-Gww**

### CODE REVIEW

File: pyuavcan/transport/can/media/pythoncan/_pythoncan.py [hunk 1]
```diff
@@ -88,6 +88,8 @@ def __init__(
 
             - Interface ``canalystii`` is described in
               https://python-can.readthedocs.io/en/stable/interfaces/canalystii.html.
+              You need download CANalyst library for python-can package or you can install python-can by:
+              ``pip3 install git+https://github.com/Cherish-Gww/python-can.git@add_canalystii_so``
```
Comment (pavel-kirienko) [inline pyuavcan/transport/can/media/pythoncan/_pythoncan.py:92]:
```suggestion
              ``pip3 install git+https://github.com/Cherish-Gww/python-can.git@add_canalystii_so``
              More info: https://github.com/UAVCAN/pyuavcan/issues/178#issuecomment-912497882
```

**Review by pavel-kirienko — CHANGES_REQUESTED** [review]
Please bump the patch version number in `pyuavcan/VERSION`.

**Review by pavel-kirienko — APPROVED** [review]
Looks good except for one missing preposition in "you need download". Should I merge this?

In the future, feel free to explicitly request a review when done using the button in the upper-right side of the page.

**Comment by Cherish-Gww** [comment]
@pavel-kirienko 
Thank you very much, I have changed annotation, forgive my poor English.
and Is there uavcan organization have a online chat to discuss issues?

**Pull request merged**